archetypes/BalancerBuySell/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Dropdown } from '@components/General/Dropdown';
+import { noDispatch, swapDefaults, useSwapContext } from '@context/SwapContext';
+import { usePool } from '@context/PoolContext';
+import { ARBITRUM, SideEnum } from '@libs/constants';
+import usePoolTokens from '@libs/hooks/usePoolTokens';
+import { tokenSymbolToLogoTicker } from '@components/General';
+import Button from '@components/General/Button';
+import { constructBalancerLink } from '@archetypes/Exchange/Summary';
+
+export default (() => {
+    const { swapState = swapDefaults, swapDispatch = noDispatch } = useSwapContext();
+    const { tokens } = usePoolTokens();
+    const { side, selectedPool } = swapState;
+    const pool = usePool(selectedPool);
```

Comment:
You probably dont need to use the entire pool here, try tokenLists and then just have a selected state